tools/system_tests.bat

```diff
@@ -0,0 +1,48 @@
+@echo off
```

Comment:
This whole file is Windows-specific.
Given that Linux supports a subset of the runtimes, do you expect to have a `.sh` file that is invoked in the Linux case?

Ideally we could have a single file (could be Python) that has a list of supported runtimes by OS and programmatically executes the appropriate list.

Have you considered this? I'm ok if that's handled in a separate PR but may be easier to do the groundwork now. It would also be less repetitive. We could even consider codegen.

Reply:
Somewhere there will be platform specific code. My thinking was there would also be a `system_tests.sh` file once we add Linux system tests, and the jobs would be generated to call the appropriate file. There is a comment to that in the .groovy file in the next PR in the chain.